pkg/tarsum/tarsum.go

```diff
@@ -29,6 +28,8 @@ import (
 	"hash"
 	"io"
 	"strings"
+
+	gzip "github.com/klauspost/pgzip"
```

Comment:
Using [github.com/klauspost/compress/gzip](https://github.com/klauspost/compress) may be better, depending on the data size. What is the typical data size here?